src/app/listener.ts

```diff
@@ -42,6 +44,15 @@ export class Listener {
 
       this.container.loggerService.info(`Loaded ${this.container.jobService.size()} jobs...`);
 
+      const slashCommandUploads = Object.entries(slashCommands).map(([key, command]) => {
+        return {
+          name: key,
+          description: command.description.substring(0, 99),
+        };
+      });
```

Comment:
This should take into account `SlashCommand#parameters`
```suggestion
      const slashCommandUploads = Object.entries(slashCommands).map(([key, command]) => {
        return {
          name: key,
          description: command.description.substring(0, 99),
          options: command.parameters,
        };
      });
```

Reply:
Yep, will do